pkg/cvo/updatepayload_test.go

```diff
@@ -0,0 +1,348 @@
+package cvo
+
+import (
+	"context"
+	"errors"
+	"testing"
+	"time"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/google/go-cmp/cmp/cmpopts"
+	configv1 "github.com/openshift/api/config/v1"
+	"github.com/openshift/library-go/pkg/verify/store"
+
+	//nolint:staticcheck // verify,Verifier from openshift/library-go uses a type from this deprecated package (needs to be addressed there)
```

Comment:
do we have a Jira ticket tracking this?

Reply:
good catch, I'll file one

Reply:
https://issues.redhat.com/browse/OCPBUGS-7479